apertium_apy/apy.py

```diff
@@ -182,7 +188,8 @@ def parse_args(cli_args=sys.argv[1:]):
     parser.add_argument('pairs_path', help='path to Apertium installed pairs (all modes files in this path are included)')
     parser.add_argument('-s', '--nonpairs-path', help='path to Apertium tree (only non-translator debug modes are included from this path)')
     parser.add_argument('-l', '--lang-names',
-                        help='path to localised language names sqlite database (default = langNames.db)', default='langNames.db')
+                        help='path to localised language names sqlite database (default = langNames.db)',
+                        default=pkg_resources.resource_filename('apertium_apy', 'langNames.db') if found is True else 'langNames.db')
```

Comment:
'langNames.db' is repeated 3 times here. Should be extracted to a global.

Reply:
I have stored it in a langNames variable now. Please check if this is how it should be.